schema/bls.mcf

```diff
@@ -15225,5 +15225,105 @@ Node: BLSSeasonallyUnadjustedEnum
 dcid: dcs:BLSSeasonallyUnadjusted
 typeOf: dcs:SeasonalAdjustmentEnum
 name: "BLSSeasonallyUnadjusted"
-description: "Indicates raw time series data without adjustments to eliminate the effect of intrayear variations"
+description: "Indicates raw time series data without adjustments to eliminate the effect of intrayear variations."
 
+Node: dcid:jobChangeEvent
+name: "jobChangeEvent"
+typeOf: schema:Property
+domain: schema:Person
+range: dcs:JobChangeEventEnum
+
+Node: dcid:JobChangeEventEnum
+name: "JobChangeEventEnum"
+description: "Describes different job change events that may happen to workers." 
+typeOf: dcs:Class
+subClassOf: Enumeration
+
+Node: dcid:Hire
+name: "Hire"
+description: "Describes a worker being hired to a company."
+typeOf: dcid:JobChangeEventEnum
+
+Node: dcid:Separation
+name: "Separation"
+typeOf: dcid:JobChangeEventEnum
+description: "An event where an employee is separated from the payroll."
+descriptionUrl: "https://www.bls.gov/jlt/jltdef.htm#4"
+
+Node: dcid:VoluntarySeparation
+name: "VoluntarySeparation"
+typeOf: dcid:JobChangeEventEnum
+description: "Quits: Employees who left voluntarily. Exception: retirements or transfers to other locations are reported with Other Separations."
```

Comment:
How about making it them fuller sentences.

"Represents employees who left voluntarily. Exceptions are retirements or transfers to ..."